gnet.go

```diff
@@ -30,7 +30,7 @@ const (
 	Shutdown
 )
 
-var defaultLogger = Logger(log.New(os.Stderr, "", log.LstdFlags))
+var defaultLogger= Logger(log.New(os.Stderr, "", log.LstdFlags))
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
File is not `gofmt`-ed with `-s` (gofmt)